src/main/content/_assets/css/home.scss

```diff
@@ -176,11 +145,6 @@ body{
 .featured-models{
     background-color: #f4f5f7;
     margin-top: 3em;
-    padding: 3em 20vw 3em 20vw;
```

Comment:
The "featured collections" section could use some padding top & bottom
<img width="1672" alt="Screen Shot 2019-09-05 at 9 00 22 AM" src="https://user-images.githubusercontent.com/37549026/64344152-e3d91700-cfbb-11e9-84a6-35a451ea12e4.png">